ipywidgets/embed.py

```diff
@@ -238,7 +238,7 @@ def escape_script(s):
     We only replace these three cases so that most html or other content
     involving `<` is readable.
     """
-    return script_escape_re.sub(r'\u003c\1', s)
+    return script_escape_re.sub(u'\u003c\1', s)
```

Comment:
I assume the second backslash here needs to be escaped? I.e. 
```suggestion
    return script_escape_re.sub(u'\u003c\\1', s)
```
This ensures that the `.sub()` method sees it as a regexp backreference (`\1`). In non-raw strings, `\1` means "byte expressed in octal (1 here)".

Reply:
Never mind this, after re-reading the initial PR, this is the wrong solution.